config/locales/doc_auth/en.yml

```diff
@@ -12,6 +12,8 @@ en:
       take_picture_retry: Retake photo
       upload_picture: Upload a photo
       use_phone: Use your phone
+    errors:
+      invalid_image_url: image data URL not formatted correctly
```

Comment:
Yes, because the result full sentence is attached to the attribute name (aka front) so that's how we get "Front image data URL not formatted correctly"

Anyways since data URLs are going away we don't need to worry about that right now 😁